esrally/racecontrol.py

```diff
@@ -23,8 +23,8 @@
 import thespian.actors
 import urllib
 
-from esrally import actor, config, exceptions, track, driver, mechanic, reporter, metrics, time, DOC_LINK, PROGRAM_NAME
-from esrally.utils import console, convert, opts
+from esrally import actor, config, exceptions, track, driver, mechanic, reporter, metrics, doc_link, PROGRAM_NAME
```

Comment:
Same comment here, there are _almost_ sorted, so we could really have them sorted.